src/test/java/org/onebusaway/alexa/AuthedSpeechletTest.java

```diff
@@ -293,6 +293,48 @@ public void help() throws SpeechletException {
     }
 
     @Test
+    public void repeat(@Mocked ObaDao obaDao) throws SpeechletException, URISyntaxException, IOException {
```

Comment:
Oh. It's redundant now. Maybe one of us just updates that one line without a PR?